src/main/java/com/greenfox/poker/service/GameStateService.java

```diff
@@ -0,0 +1,57 @@
+package com.greenfox.poker.service;
+
+
+import com.greenfox.poker.gamestates.Betting;
+import com.greenfox.poker.gamestates.BettingCycleInEachRound;
+import com.greenfox.poker.gamestates.Flop;
+import com.greenfox.poker.gamestates.River;
+import com.greenfox.poker.gamestates.ShowDown;
+import com.greenfox.poker.gamestates.Turn;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class GameStateService {
+
+  GameService gameService;
+  BettingCycleInEachRound bettingCycleInEachRound = new BettingCycleInEachRound(gameService);
+
+  @Autowired
+  public GameStateService(GameService gameService) {
+    this.gameService = gameService;
+  }
+
+
+  private void bettingRound(long gameStateId){
+    Betting betting = new Betting(gameService);
+    betting.initForBettingGameState(gameStateId);
+    bettingCycleInEachRound.initBettingCycle(gameStateId);
+    bettingCycleInEachRound.bettingCycle(gameStateId);
```

Comment:
You are calling the same two functions at the end of all your functions here.
This is duplication.
There should be a better way to solve this.